shared/style/dialer/dialer.css

```diff
@@ -9,6 +16,7 @@ html,body {
   margin: 0;
   color: white;
   background-color: white;
+  -moz-user-select: none;
```

Comment:
This shouldn't be needed given https://github.com/mozilla-b2g/gaia/pull/27159/files#r22540359